plugins/inputs/kapacitor/README.md

```diff
@@ -96,6 +104,36 @@ The total number of Kapacitor tasks.
 
 ---
 
+### kapacitor_alert
+The `kapacitor_alert` measurement stores fields with information related to
+[Kapacitor alerts](https://docs.influxdata.com/kapacitor/v1.5/working/alerts/).
+
+#### notification-dropped
+The number of alert or event handlers that failed.
+
+#### primary-handle-count
+The number of alerts counted for the primary event handler.
+
+#### secondary-handle-count
+The number of alerts counted for the secondary event handler.
+
+---
+
+### kapacitor_cluster
+The `kapacitor_cluster` measurement stores fields with information related to
+[Kapacitor clusters](https://docs.influxdata.com/enterprise_kapacitor/v1.5/cluster-management/). Kapacitor Enterprise deduplicates alert data to prevent duplicate alert notifications from being sent.
```

Comment:
The cluster metrics are not specific to alert deduplication rather the reflect the ability of Kapacitor nodes to communicate with one another. Specifically these metrics track the gossip communication between the Kapacitor nodes.  https://docs.influxdata.com/enterprise_kapacitor/v1.5/administration/configuration/#cluster-communications